packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1479,6 +1499,14 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
   TextSelection get textSelection => _textSelection;
   TextSelection _textSelection;
 
+  /// The count of children of a scroll node which contribute semantics.
+  int get scrollChildren => _scrollChildren;
```

Comment:
nit: `scrollChildren` sounds like the getter returns the children, while in fact it returns count. How about call it `scrollChildCount`?